examples/sum_rec/main.hvm

```diff
@@ -1,8 +1,8 @@
 @main = a
-  & @sum ~ (28 (0 a))
+  & @sum ~ (24 (0 a))
 
 @sum = (?(((a a) @sum__C0) b) b)
 
 @sum__C0 = ({c a} ({$([*2] $([+1] d)) $([*2] $([+0] b))} f))
-  &! @sum ~ (a (b $(:[+] $(e f))))
```

Comment:
Not sure if we should fix these manually here or coordinate with the bend team to use the new syntax.

Reply:
probably both; although I think the latter may already have been done?